bolt/runtime/hugify.cpp

```diff
@@ -36,7 +36,6 @@ static void getKernelVersion(uint32_t *Val) {
   // release should be in the format: %d.%d.%d
   // major, minor, release
   struct UtsNameTy UtsName;
-  int Ret = __uname(&UtsName);
```

Comment:
Can't remove setting UtsName

Reply:
I just updated it. Could you re-review it please?